src/components/veNft/VeNftOwnedTokensSection.tsx

```diff
@@ -27,26 +28,34 @@ export default function OwnedNFTSection({
   const hasOverflow = Boolean(primaryTerminalCurrentOverflow?.gt(0))
 
   return (
-    <div style={{ ...shadowCard(theme), padding: 25, marginBottom: 10 }}>
+    <>
```

Comment:
**question**: does it still work if we remove the wrapping fragment?

Reply:
it doesn't, i could use a div